src/docs/sphinx/dev_guide/updating_tpls.rst

```diff
@@ -172,105 +186,119 @@ Updating TPLs
+
+From the top-level Axom directory in a local clone of the repo, run the following command
+to build all TPLs for all existing compiler specs on the platform you are currently on::
 
 $ ./scripts/llnl_scripts/build_tpls.py -d local/install/path
 
 where ``local/install/path`` is a directory location where you want the 
 libraries to be installed.
 
-It will output whether the TPL install succeeded and, 
+The TPL build script will output whether each TPL install succeeded and, 
 subsequently, whether an Axom build against the TPL install succeeded.
 
+.. note:: When Spack runs, you may see what looks like an error related to ``axom@develop``
+          being unable to download. This is not an actual error, but a "feature" of how 
+          Spack reports what it's doing, and can be ignored.
+
 Running the script produces new host-config files (i.e., CMake cache files) 
-that you can use to build and test Axom with the installation, if issues
-arise. The generated host-config files will be located in the top-level Axom
+that you can use to build and test Axom against the installation for development or 
+if issues arise. The generated host-config files will be placed in the top-level Axom
 directory of your local clone of the repo. If any changes to Axom code are 
-needed to work with the TPL update(s), make the changes and test them.
+needed to work with the TPL update(s), make the changes there and test them.
 
-.. note:: You can build a subset of TPLs for a platform, by using
-          the ``uberenv.py`` script in the top-level Axom directory.
-          For example:: 
+.. note:: You can build a subset of TPLs for a platform, by using the ``uberenv.py``
+           script in the top-level Axom directory. For example:: 
 
             python3 ./scripts/uberenv/uberenv.py --prefix /my/tpl/path --spec clang@10.0.0~cpp14+devtools+mfem+c2c
 
           will build the TPLs for the clang 10.0.0 compiler, install them
           to the ``/my/tpl/path`` directory, and generate a host-config file
           that you can use to build Axom and its tests. Please see the
-          ``scripts/spack/specs.json`` file for a current list of tested specs. 
+          ``scripts/spack/specs.json`` file for a current list of TPL specs
+          we use for GitLab CI testing.
 
 
 Shared Third-party Library Installation Steps
 ---------------------------------------------
 
-The following instructions describe how to install local copies of Axom
-TPLs on Livermore Computing (LC) platforms and recreate our Docker containers
+The following instructions describe how to install copies of Axom TPL builds
+on Livermore Computing (LC) platforms and recreate our Docker containers
 with a new set of TPLs. Typically, this process is followed when you want to 
-update one or more TPLs which Axom depends on. After they are built and
-the required changes are merged into develop, they will be available for
+update one or more TPLs on which Axom depends. After they are built and
```

Comment:
Possibly:
```suggestion
update one or more TPLs. After they are built and
```
They're our TPLs.  We don't have to say again that we depend on them.

Reply:
Thank you. Fixed.